Resources/public/js/views/services/ez-roleserversideviewservice.js

```diff
@@ -295,5 +299,19 @@ YUI.add('ez-roleserversideviewservice', function (Y) {
                 }
             });
         },
+
+        /**
+         * Refreshes the role view, showing the assignments tab.
+         *
+         * @method _refreshAssignmentsTab
+         * @protected
+         * @param {Object} data
+         */
+        _refreshAssignmentsTab: function (data) {
+            this.get('app').set('loading', true);
+            this.load(Y.bind(function () {
+                this.get('app').navigateTo('adminRole', {uri: 'pjax/role/view/' + data.roleId + '/ez-tabs-content'});
```

Comment:
The 2nd paragraph isn't clear to me. Why/how are there two requests? Is the `set('loading', true)` an ajax request?